src/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -43,43 +40,35 @@ const AdminDashboard = () => {
 
         if (token) {
             getUserDetails()
+            dispatch(getRooms(t))
         }
-    }, [token, userId, dispatch])
+    }, [token, userId, dispatch, t])
 
-    const preventDefault = (event: React.SyntheticEvent) =>
-        event.preventDefault()
+    const onFormSubmit = (values, isEditing) => {
+        const { roomName, hostName, destinations, roomId } = values
```

Comment:
`roomId` doesn't exist in form values. So you call edit room api with an undefined id

Reply:
Well seen man, I was thinking on how to handle this. Do you think an hidden field within the form would be the cleaner way to treat this ?

Reply:
maybe by using state